docroot/modules/custom/app_module/src/AppPathManager.php

```diff
@@ -84,6 +84,34 @@ class AppPathManager implements AppPathManagerInterface, CacheDecoratorInterface
    */
   protected $lookupMap = [];
 
+  /**
+   * Pre loaded path lookups.
+   *
+   * @var array
+   */
+  protected $preloadedPathLookups = [];
```

Comment:
Somehow all these properties are used in /core/modules/path_alias/src/AliasManager.php. Both the classes aliasManager and appPathManager are implementing the same interface and these properties are used in clearCache method. Not sure if I removed this from appPathManager service, it may impact somewhere in clearing cache.

Reply:
Ok, based on our discussion you will just remove the instances in clear cache where these were being set.